### PR TITLE
Added sample policy to check secret token type

### PR DESCRIPTION
**Pull request by siddhikhapare**
## Related Issue(s)
#819 

## Description
This policy aims to address  security problems by auditing to find and fix issues with certain types of long-lasting service accounts that could be risky.

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: other/audit-secret-token-type/audit-secret-token-type.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - name: validate-service-account-token-secrets
+    match:
+      any:
+      - resources:
+          kinds:
+          - Secret
+          namePattern: ".*" 
+          apiGroups: [""]
+          apiVersions: ["v1"]
+          resourceNames: [""]
```
Comment (chipzoller) [inline other/audit-secret-token-type/audit-secret-token-type.yaml:33]: Where do you get these from, ChatGPT? This isn't a valid policy at all and you would have known that if you tried to even create it, which also means you haven't personally tested it.

**Review by chipzoller — CHANGES_REQUESTED** [review]
The policy isn't correct.

File: other/audit-secret-token-type/audit-secret-token-type.yaml (hunk 1 above)
Comment (siddhikhapare) [reply other/audit-secret-token-type/audit-secret-token-type.yaml:33]: @chipzoller  I used chatgpt for reference but I thought using secrets  pattern will be helpful to get all resources such as pod and deployment to test type of that ```type: kubernetes.io/service-account-token```. I tested my files of resoure.yaml where I referred from various blogs  https://medium.com/@th3b3ginn3r/understanding-service-accounts-in-kubernetes-e9d2abe19df8 and https://spacelift.io/blog/kubernetes-secrets but I found error regarding this policy.

Comment (siddhikhapare) [reply other/audit-secret-token-type/audit-secret-token-type.yaml:33]: @chipzoller I tried with ```resource.object.type``` it's also not working. I will try with CEL.

Comment (chipzoller) [reply other/audit-secret-token-type/audit-secret-token-type.yaml:33]: Please do not use ChatGPT, or any other AI tool for that matter, to tell you what policies to write. Not only is it wrong, but it's just fantasy. Please use the docs, existing samples, and `kubectl explain` to understand the fields available when authoring policies. You do not need to resort to CEL to implement this simple policy.

Comment (siddhikhapare) [reply other/audit-secret-token-type/audit-secret-token-type.yaml:33]: @chipzoller Sure. Thank you for your feedback. I will look into it.

**Pull request closed without merging**